src/main/java/it/eng/idsa/businesslogic/service/impl/RejectionMessageServiceImpl.java

```diff
@@ -36,25 +35,21 @@ public class RejectionMessageServiceImpl implements RejectionMessageService{
 	
```

Comment:
Can we refactor this method, to have only one createRejectionMessage with 2 parameters, RejectionType and "originalMessage"?
Since all methods looks the same, only difference is RejectionType

Reply:
I just realised that too whilst adding the rejection in a previous issue you addressed. Will be done right away!